com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/formatter/StringObjectFormatter.java

```diff
@@ -26,7 +26,7 @@
 
 public class StringObjectFormatter extends ObjectFormatter implements IValueFormatter {
     public static final String MAX_STRING_LENGTH_OPTION = "max_string_length";
-    private static final int DEFAULT_MAX_STRING_LENGTH = 100;
+    private static final int DEFAULT_MAX_STRING_LENGTH = 200;
```

Comment:
Can I understand why 100 -> 200?

Reply:
the default length is the length most users saw, the 100 is too small for normal cases like connections string, path, I would advise to use 256.